docs/05. module guide/06. blocks_and_widgets.md

```diff
@@ -38,7 +38,7 @@ private $record;
 Then we define our (private) variables. In our case we'll use an array to save the record with the article we will be viewing.
 
 ```
-public function execute()
+public function execute(): void
 {
  // call the parent 
```

Comment:
The indentation of the entire php code block does not seem valid.

Reply:
I just changed is online and it's the same indent as in the core.,
Only added the :void

Reply:
My mistake, forgot to download the example module and just checked the code with the documentation by copy pasting it and creating a new MiniBlog module by myself.

Reply:
Never mind, we will change this after your PR